app/stats.go

```diff
@@ -5,10 +5,11 @@ import (
 	"github.com/codegangsta/cli"
 )
 
-func LsStats() cli.Command {
+// JournalFlush flush operations since last flush
+func JournalFlush() cli.Command {
```

Comment:
Why we want to call it JournalFlush()? Flush is an implicit action of calling journal. Just do Journal() sounds better because it's the main purpose of this command - showing the journal.